ui/components/ui/user-nav/user-nav.tsx

```diff
@@ -24,11 +24,14 @@ import {
   TooltipProvider,
   TooltipTrigger,
 } from "@/components/ui/tooltip/tooltip";
-import { UserProfileProps } from "@/types";
+import { useStore } from "@/hooks/use-store";
+import { useUser } from "@/hooks/use-user";
 
 import { Button } from "../button/button";
 
-export const UserNav = ({ user }: { user?: UserProfileProps }) => {
+export const UserNav = () => {
+  const userStore = useStore(useUser, (x) => x);
```

Comment:
I would avoid using useStore. It relies on localStorage to store user info, which I don’t think is a good practice. I don’t really see the need for it if we can just use useSession instead